src/trie.ts

```diff
@@ -5,9 +5,15 @@ import {
 	TrieRegExp,
 	ProcessedSynonyms,
 	IteratedTrieValue,
+	ReservedWords,
 } from './types';
 
-const RESERVED_WORDS = new Set(['$word', '$synonymTrie', '$value']);
+const RESERVED_WORDS = new Set<string>();
```

Comment:
I still create that Set to make sure no funny guy tries to change the ReservedWords enum to break this package